test/ast.wisp

```diff
@@ -1,5 +1,5 @@
 (ns wisp.test.ast
-  (:require [wisp.test.util :refe [is thrown?]]
+  (:require [wisp.test.util :refer [is thrown?]]
```

Comment:
…Apparently importing macros does nothing after all?

Reply:
Yeah it will import all macros regardless. Intention was to only import ones been referred, but that never made it.